src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -25,6 +29,10 @@
  */
 public class CitizenData
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') class "CitizenData" has 50 methods, which is greater than the 35 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1448)